src/lava/magma/core/run_configs.py

```diff
@@ -67,42 +67,60 @@ def select(self,
         pass
 
 
-# ToDo: This is only a minimal RunConfig that will select SubProcModel if there
-#       is one and if not selects a PyProcessModel with the correct tag.
-#       Needs to be modified in future releases to support more complicated
-#       sync domains, @requires (example: GPU support), and to select
-#       LeafProcModels of type other than PyProcessModel.
 class Loihi1SimCfg(RunConfig):
     """Run configuration selects appropriate ProcessModel -- either
-    SubProcessModel for a Hierarchical Process or else a PyProcessModel for a
-    standard Process. The appropriate PyProcessModel is selected based @tag(
-    'floating_pt') or @tag('fixed_pt'), for floating point precision or Loihi
-    bit-accurate fixed point precision respectively"""
+    `SubProcessModel` for a hierarchical Process or else a `PyProcessModel`
+    for a standard Process.
 
-    def __init__(self, custom_sync_domains=None, select_tag='floating_pt',
-                 select_sub_proc_model=False):
+    If any `SubProcessModel`s are available, then they will be returned first.
```

Comment:
A bit below, you say that priority is given first exception_proc_model_map. I think this should be also clearified here in docstring

Reply:
Rearranged the docstring to reflect this.